src/tabs/ux-tab-theme.css

```diff
@@ -0,0 +1,34 @@
+styles.tab {
+  display: inline-block;
+  text-transform: uppercase;
+  cursor: pointer;
+  opacity: 0.7;
+  padding: 14px 12px 20px 12px;
+  text-align: center;
+  color: ${foreground || $design.accentForeground};
+  background-color: ${background || $design.accent};
```

Comment:
I would not base the background on the accent.

Try leaving this as either transparent or set it to the primary color.